api/src/server.ts

```diff
@@ -82,7 +84,7 @@ export async function createServer(): Promise<http.Server> {
 	});
 
 	const terminusOptions: TerminusOptions = {
-		timeout: 1000,
+		timeout: env['SERVER_SHUTDOWN_TIMEOUT'],
```

Comment:
should we cast this to a number and perhaps do some sanity checking against `Inifinity` or `NaN`?